build/variables.yml

```diff
@@ -65,25 +65,25 @@ variables:
   # AzureStorageAccountName:
 
   # Solution to build
-  SolutionFileName: ApplicationTemplate.sln
+  SolutionName: ApplicationTemplate
 
   # Paths to file
-  InfoPlistPath: $(Build.SourcesDirectory)/src/app/ApplicationTemplate.iOS/Info.plist
-  AndroidManifestPath: $(Build.SourcesDirectory)/src/app/ApplicationTemplate.Droid/Properties/AndroidManifest.xml
+  InfoPlistPath: '$(Build.SourcesDirectory)/src/app/$(SolutionName).Mobile/iOS/Info.plist'
+  AndroidManifestPath: '$(Build.SourcesDirectory)/src/app/$(SolutionName).Mobile/Android/AndroidManifest.xml'
 
-  # Pool names
-  windowsPoolName: 'windows 2022'
+  # Virtual machine images
+  windowsVMImageName: 'windows-2022'
```

Comment:
I would suggest to specify in the name that its hosted so its clear for everyone that we don't use our self-hosted agents. 

`windowsHostedAgentImage`:
`macOSHostedAgentImage`:

Reply:
All right make sense!